### PR TITLE
Change the background color of the form in the contact section

### LINKED ISSUE #165: Change the background color of the form in the contact section
- Currently, the background color of the form in the contact section is different than the default section background color
- Change the background color of the form to the background color in the default section `second-section`

image

### PR DESCRIPTION
**Pull request by Rishav1707**
## Purpose

The purpose of this Pull Request is to fix #165 <issue-number>

## Goals

- This feature will fix the background of the contact section.

## Approach

- Adding the background color property in the contact section class name.

### Before
206942712-f9ac8e57-981d-482a-861c-132524e11536
### After
Screenshot 2022-12-13 073917

### Preview Link

https://pr-177-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
image

There is a more difference between the form bg and second section bg. There is a class for the second section and inserted a bg color with a blur effect find it and add them @Rishav1707

File: dark/index.html [hunk 1]
```diff
@@ -212,18 +212,18 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
      <section id="contact"> 
       <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3> 
-      <div class="formContainer my-3 bg-dark p-5 rounded-3">
+      <div class="formContainer my-3 p-5 rounded-3">
         <form class="m-auto">
           <div class="mb-3">
-            <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
+            <input type="text" class="form-control text-white" placeholder="Enter Your Name"
```
Comment (YoshithaRathnayake) [inline dark/index.html:218]: Don't remove the bg-dark of the inputs @Rishav1707 

```suggestion
            <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 2]
```diff
@@ -212,18 +212,18 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
...
-      <div class="formContainer my-3 bg-dark p-5 rounded-3">
+      <div class="formContainer my-3 p-5 rounded-3">
         <form class="m-auto">
           <div class="mb-3">
             <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
               id="exampleInputPassword1" />
           </div>
           <div class="mb-3">
-            <input type="email" class="form-control bg-dark text-white" placeholder="Enter Your Email" id="email"
+            <input type="email" class="form-control text-white" placeholder="Enter Your Email" id="email"
```
Comment (YoshithaRathnayake) [inline dark/index.html:222]:
```suggestion
            <input type="email" class="form-control bg-dark text-white" placeholder="Enter Your Email" id="email"
```

File: dark/index.html [hunk 3]
```diff
@@ -212,18 +212,18 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
...
               aria-describedby="emailHelp" />
           </div>
           <div class="mb-3">
-            <textarea name="message" id="message" row="20" col="20" class="form-control bg-dark text-white"
+            <textarea name="message" id="message" row="20" col="20" class="form-control text-white"
```
Comment (YoshithaRathnayake) [inline dark/index.html:226]:
```suggestion
            <textarea name="message" id="message" row="20" col="20" class="form-control bg-dark text-white"
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by YoshithaRathnayake** [comment]
> image
> 
> There is a more difference between the form bg and second section bg. There is a class for the second section and inserted a bg color with a blur effect find it and add them @Rishav1707

How about this @Rishav1707

**Comment by Rishav1707** [comment]
> > image
> > There is a more difference between the form bg and second section bg. There is a class for the second section and inserted a bg color with a blur effect find it and add them @Rishav1707
> 
> How about this @Rishav1707

ok I will look up to this and report to you soon.

**Comment by Rishav1707** [comment]
> > image
> > There is a more difference between the form bg and second section bg. There is a class for the second section and inserted a bg color with a blur effect find it and add them @Rishav1707
> 
> How about this @Rishav1707

Done @YoshithaRathnayake

File: dark/assets/css/styles.css [hunk 4]
```diff
@@ -1097,7 +1097,9 @@ html {
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  background-color: #060606;
+  box-shadow: 0 0 50px rgba(0, 0, 0, 1);
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1100]: Let's remove the box shadow @Rishav1707 
```suggestion
```

**Comment by Rishav1707** [comment]
@YoshithaRathnayake Check the website once again.

File: dark/assets/css/styles.css [hunk 5]
```diff
@@ -1097,6 +1097,8 @@ html {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  background-color: rgba(0, 0, 0, 0.8);
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1100]: Don't reduce the bg color @Rishav1707 
```suggestion
  background-color: rgba(0, 0, 0, 0.9);
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by YoshithaRathnayake** [comment]
> image
> 
> There is a border like thing in the top of the form background, can you try to remove it? @Rishav1707

You have to fix this @Rishav1707

**Comment by Rishav1707** [comment]
> > image
> > There is a border like thing in the top of the form background, can you try to remove it? @Rishav1707
> 
> You have to fix this @Rishav1707

But how?
Because I don't think that the top of the form is much darker.
I applied the same background to the contact section from the second section.

**Comment by YoshithaRathnayake** [comment]
> > > image
> > > There is a border like thing in the top of the form background, can you try to remove it? @Rishav1707
> > 
> > 
> > You have to fix this @Rishav1707
> 
> But how? Because I don't think that the top of the form is much darker. I applied the same background to the contact section from the second section.

I also don't know about that but its ugly can you try to add all the things in that bg color to a one section and add that bg color to the created section @Rishav1707

**Comment by Rishav1707** [comment]
> > > > image
> > > > There is a border like thing in the top of the form background, can you try to remove it? @Rishav1707
> > > 
> > > 
> > > You have to fix this @Rishav1707
> > 
> > 
> > But how? Because I don't think that the top of the form is much darker. I applied the same background to the contact section from the second section.
> 
> I also don't know about that but its ugly can you try to add all the things in that bg color to a one section and add that bg color to the created section @Rishav1707

I don't understand what you are saying.

**Comment by YoshithaRathnayake** [comment]
- Create a section first
- Add all the content from the about section to it
- Create a css class for that section
- And add that background color part to that css class

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
- Create a section first
- Add all the content from the about section to it
- Create a css class for that section
- And add that background color part to that css class

File: dark/assets/css/styles.css [hunk 6]
```diff
@@ -1126,4 +1131,4 @@ html {
   border-radius: 2rem;
   font-size: 1.1rem;
   transition: 0.2 ease;
-}
+}
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1134]: Add the end of the line @Rishav1707 
```suggestion
}

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 7]
```diff
@@ -1097,6 +1102,8 @@ html {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  background-color: rgba(0, 0, 0, 0.9);
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1105]: Remove this bg color and blur property @Rishav1707

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 8]
```diff
@@ -190,9 +190,9 @@ <h1>
       </div>
     </center>
   </section>
-  
-  <section class="second-section">
-    <!-- About Section -->
+
+  <!-- <section class="second-section">
```
Comment (YoshithaRathnayake) [inline dark/index.html:194]: Good job @Rishav1707
Can you remove the new section created and add all the content to that second-section.
And add that bg color to it

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Yeah now I can take a decision you can stay with the early change that demo-section @Rishav1707

**Comment by Rishav1707** [comment]
OK @YoshithaRathnayake

File: dark/index.html [hunk 9]
```diff
@@ -281,11 +311,58 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+    const sidebar = document.querySelector(".sidebar");
+
+    menu_toggle.addEventListener("click", () => {
+      menu_toggle.classList.toggle("is-active");
+      sidebar.classList.toggle("is-active");
+    });
+  </script>
+
+</body>
 </html>
```
Comment (YoshithaRathnayake) [inline dark/index.html:368]: Add the end of the line @Rishav1707 
```suggestion
</html>

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
It's repeated the same one two times, fix it @Rishav1707 

image

File: dark/assets/css/styles.css (hunk 6 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1127]: Do this change @Rishav1707 
You still didn't done it, don't resolve the conversations again

```suggestion
}

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by Rishav1707** [comment]
@YoshithaRathnayake Review my changes...

File: dark/index.html [hunk 10]
```diff
@@ -7,7 +7,8 @@
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-  <meta name="description" content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 14 years old. I am studing in St. Anthony's College, Kandy" />
+  <meta name="description"
```
Comment (YoshithaRathnayake) [inline dark/index.html:10]: You have formatted the code, please undo that @Rishav1707

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 10 above)
Comment (Rishav1707) [reply dark/index.html:10]: ok

**Comment by Rishav1707** [comment]
@YoshithaRathnayake Please review my changes...

File: dark/index.html [hunk 11]
```diff
@@ -243,6 +252,8 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
 
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
 
+  <!-- Nav-bar Scroll Animation -->
```
Comment (YoshithaRathnayake) [inline dark/index.html:255]: Remove this @Rishav1707 
```suggestion

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 12]
```diff
@@ -243,6 +252,7 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
 
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
 
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:255]: Remove this line @Rishav1707 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
image
**_Error!_**

**Pull request closed without merging**